src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -678,12 +682,68 @@ void GraphOptimizer::FuseMultiplyAndAdd(Graph& graph) {
     }
 }
 
-void GraphOptimizer::MergeConvertAndScaleShift(Graph& graph) {
+void GraphOptimizer::MergeEltwiseAndConvert(Graph& graph) {
+// The pass is required on arm platforms only
+#if !defined(OPENVINO_ARCH_ARM) && !defined(OPENVINO_ARCH_ARM64)
+    return;
+#endif
+    auto& graphNodes = graph.GetNodes();
+
+    auto parent = graphNodes.begin();
+    while (parent != graphNodes.end()) {
+        CPU_GRAPH_OPTIMIZER_SCOPE(MergeEltwiseAndConvert);
+        auto parentNode = *parent;
+        if (parentNode->getType() != Type::Eltwise) {
+            parent++;
+            continue;
+        }
+
+        const auto& childEdges = parentNode->getChildEdges();
+        if (childEdges.size() != 1) {
+            parent++;
+            continue;
+        }
+
+        const auto edge = childEdges[0].lock();
+        auto childNode = edge->getChild();
+        if (childNode->getType() != Type::Convert) {
+            parent++;
+            continue;
+        }
+
+        if (!one_of(childNode->getOriginalOutputPrecisionAtPort(0),
+                    ov::element::i8,
+                    ov::element::u8,
+                    ov::element::f16,
+                    ov::element::bf16,
+                    ov::element::f32)) {
+            parent++;
+            continue;
+        }
+
+        auto parentEdges = childNode->parentEdges;
+        for (auto& parentEdge : parentEdges) {
+            auto p_edge = parentEdge.lock();
+            if (p_edge->getParent()->getType() == Type::Eltwise) {
+                continue;
+            }
+            graph.RemoveEdge(p_edge);
+        }
```

Comment:
Seems like this code is already part of graph.DropNode(childNode);

Reply:
It looks so, removed this code